ocs_ci/ocs/node.py

```diff
@@ -46,21 +46,27 @@ def get_node_objs(node_names=None):
     return nodes
 
 
-def get_typed_nodes(node_type='worker', num_of_nodes=None):
+def get_typed_nodes(node_type=constants.WORKER_MACHINE, num_of_nodes=None, ocs_node=False):
     """
     Get cluster's nodes according to the node type (e.g. worker, master) and the
     number of requested nodes from that type
 
     Args:
         node_type (str): The node type (e.g. worker, master)
         num_of_nodes (int): The number of nodes to be returned
+        ocs_node (bool): If true, then chooses only ocs nodes. Else if false,
+            it will choose a node not necessarily from the ocs nodes.
 
     Returns:
         list: The nodes OCP instances
 
     """
+    node_list = get_node_objs()
+    if ocs_node and node_type == constants.WORKER_MACHINE:
```

Comment:
are we sure we want an OCS machine in all cases a worker is needed?
@nehaberry @vavuthu @PrasadDesala please suggest, thanks

Reply:
IMO, the function should be generic to handle both master and worker nodes.
In some tests we get master nodes as well.

Reply:
Maybe it will be better to implement a new function `def get_ocs_nodes`